src/mds/CDir.cc

```diff
@@ -2922,8 +2935,6 @@ void CDir::scrub_initialize()
     }
   }
   scrub_infop->directory_scrubbing = true;
-
-  assert(scrub_local()); // TODO: handle failure
```

Comment:
because I don't want to call scrub_local() twice for each dirfrag. Once at here, once in CInode::validate_disk_state